src/services/pollService.js

```diff
@@ -61,8 +63,8 @@ export const createPoll = ({
             clearInterval(pollId);
             actions.onSuccess(txHash);
             return res;
-          } 
-          tryCount =+ 1;
+          }
+          tryCount = +1;
```

Comment:
I'm curious about the '+' operator here. Is it to coerce to a number type? If so, describing the number as a literal already does that.

Reply:
not sure how that happened. linter must have done it.

should be += right?

Reply:
tryCount += 1;